gammapy/datasets/tests/test_flux_points.py

```diff
@@ -24,18 +31,18 @@ def dataset():
     path = "$GAMMAPY_DATA/tests/spectrum/flux_points/diff_flux_points.fits"
     table = Table.read(make_path(path))
     table["e_ref"] = table["e_ref"].quantity.to("TeV")
+    gti = GTI.create(start=0 * u.s, stop=30 * u.min)
     data = FluxPoints.from_table(table, format="gadf-sed")
+    data.gti = gti
```

Comment:
Same comment as above: the `.gti` attribute should be added to `__init__` and documented.